ideascube/views.py

```diff
@@ -25,23 +25,49 @@
 from ideascube.library.models import Book
 from ideascube.mediacenter.models import Document
 
+# Do not import directly Catalog as we want to mock it for test.
+from ideascube.serveradmin import catalog as catalog_mod
```

Comment:
You're using `catalog_mod` here to avoid clashing between the `catalog` module and your `catalog` variable.

Why not directly import `from ideascube.serveradmin.catalog import Catalog` since that's the only thing you ever seem to use from the module?